www/include/Administration/parameters/remote/formRemote.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+use Security\Encryption;
+
+if (!isset($centreon)) {
+    exit();
+}
+
+/*
+ * Set encryption parameters
+ */
+require_once _CENTREON_PATH_ . "/src/Security/Encryption.php";
+$firstKey = @include _CENTREON_PATH_ . "/.env.local.php";
+if (empty($firstKey)) {
+    exit();
+}
```

Comment:
`$firstKey` is a bit confusing since `.env.local.php` should be an array
```suggestion
if (file_exists(_CENTREON_PATH_ . '/.env.local.php')) {
    $localEnv = @include _CENTREON_PATH_ . '/.env.local.php';
}

if (empty($localEnv) || !isset($localEnv['APP_SECRET'])) {
    exit();
}
```